### PR TITLE
css of header and navbar

### PR DESCRIPTION
**Pull request by Himeurfarid**

### CODE REVIEW

File: frontend/src/components/NewsFeed.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ function NewsFeed({ communicationPlans }) {
           src="https://remeng.rosselcdn.net/sites/default/files/dpistyles_v2/ena_16_9_extra_big/2018/12/10/node_20755/530170/public/2018/12/10/B9717886568Z.1_20181210094232_000%2BGERCIOLFT.1-0.jpg?itok=t6jVgxVZ1544431361"
           alt="imagecontent"
         />
-        <div>
+        {/* <div>
```
Comment (rocambille) [inline frontend/src/components/NewsFeed.jsx:44]: dead code

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/NewsFeed.jsx (hunk 1 above)
Comment (Himeurfarid) [reply frontend/src/components/NewsFeed.jsx]: i remove this

File: frontend/src/components/NewsFeed.jsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ function NewsFeed({ communicationPlans }) {
               />
             );
           })}
-        </div>{" "}
+        </div>
         */
```
Comment (rocambille) [inline frontend/src/components/NewsFeed.jsx:55]: pretty sure the `*/` should be removed as well ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/NewsFeed.jsx (hunk 2 above)
Comment (Himeurfarid) [reply frontend/src/components/NewsFeed.jsx:55]: TY

Comment (Himeurfarid) [reply frontend/src/components/NewsFeed.jsx]: Its'ok

**Pull request merged**